Doug/Items/EquipmentItem.cs

```diff
@@ -30,18 +30,32 @@ public class EquipmentItem : Item
         public int Dodge { get; set; }
         public int Hitrate { get; set; }
         public int AttackSpeed { get; set; }
+        public int CooldownReduction { get; set; }
+        public int Pierce { get; set; }
         public int Resistance { get; set; }
         public int Luck { get; set; }
         public int Agility { get; set; }
         public int Strength { get; set; }
         public int Constitution { get; set; }
         public int Intelligence { get; set; }
         public int HealthRegen { get; set; }
+        public int FlatHealthRegen { get; set; }
         public int EnergyRegen { get; set; }
+        public int FlatEnergyRegen { get; set; }
+
+        public double LuckFactor { get; set; }
+        public double AgilityFactor { get; set; }
+        public double StrengthFactor { get; set; }
+        public double ConstitutionFactor { get; set; }
```

Comment:
make sure they are integers : 5 would mean an increase of 5%

Reply:
Alright, this is fixed. Also removed the redundant methood SumDoubles I made, so this is a bit less stupid